bin/config_converter

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python3
```

Comment:
Can we omit this file, and just run config_converter.py directly?

Reply:
I think so - you'd have to run something like `python3 -m config_converter.config_converter` and then all the arguments like file path etc. I just thought this would be neat/easier, but I can remove it.